waspc/src/Wasp/Package.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Wasp.Package
```

Comment:
Renamed to `NodePackageFFI`. I didn't change how the variables are named inside the module, it all just refers to "package" since the module name gives the context.

> Are we ok with tying this just to Node?

I think tying it to node is better for now, especially since the code relies heavily on them being node programs. If we end up wanting to run code from other languages we can just rename the module again. #